.sandstorm/usr/share/gitweb/publish.cgi

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+echo 'Content-type: text/plain'
+echo ''
+echo '(re)publishing the gw-pages branch...'
+exec 2>&1
+cd /var/repo.git
+echo 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa' \
```

Comment:
What do all these `a`s do? Can you add a comment explaining them?

Reply:
Good question. I added a comment. Let me know whether it's clear enough.